tests/fixtures/mockbin/mockbin

```diff
@@ -152,4 +153,54 @@ elif prog in ['koji', 'brew']:
             print('BUILD: {}\nState: COMPLETE'.format(args.pkg))
 
 elif prog in ['copr-cli']:
-    print({})
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand')
+
+    parser_build = subparsers.add_parser('get-package')
+    parser_build.add_argument('project')
+    parser_build.add_argument('--name')
+    parser_build.add_argument('--with-all-builds', action='store_true')
+
+    parser_build = subparsers.add_parser('build')
+    parser_build.add_argument('project')
+    parser_build.add_argument('pkg')
+    parser_build.add_argument('--nowait', action='store_true')
+
+    parser_build = subparsers.add_parser('list')
+    parser_build.add_argument('project')
+
+    parser_build = subparsers.add_parser('fork')
+    parser_build.add_argument('src')
+    parser_build.add_argument('dest')
+
+    parser_build = subparsers.add_parser('create')
+    parser_build.add_argument('project')
+    parser_build.add_argument('--description')
+    parser_build.add_argument('--chroot')
+    parser_build.add_argument('--appstream')
+    parser_build.add_argument('--delete-after-days')
+
+    parser_build = subparsers.add_parser('modify')
+    parser_build.add_argument('project')
+    parser_build.add_argument('--delete-after-days')
+
+    parser_build = subparsers.add_parser('edit-chroot')
+    parser_build.add_argument('project-chroot')
+    parser_build.add_argument('--repos')
+    parser_build.add_argument('--packages')
+    parser_build.add_argument('--modules')
+
+    parser_build = subparsers.add_parser('watch-build')
+    parser_build.add_argument('build')
+
+    args = parser.parse_args()
+
+    if args.subcommand == 'get-package':
+        json_data = json.loads('{"builds": [{"state": "succeeded", "source_package": {"version": "0.0.1"}}]}')
+        print(json.dumps(json_data, indent=2))
```

Comment:
why loads/dumps here? you could either take the string you have verbatim and print it (it is valid json), or have a "real" python dict and dumps it.